cms/static/js/views/paging_header.js

```diff
@@ -40,32 +46,40 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
                     if (this.view.collection.sortDirection === 'asc') {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added ascending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s ascending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, filtered by {assetType},' +
+                            ' sorted by {sortName} ascending'
+                        );
                     } else {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added descending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s descending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, filtered by {assetType},' +
+                            ' sorted by {sortName} descending');
                     }
                     asset_type = this.filterNameLabel();
-                }
-                else {
+                } else {
                     if (this.view.collection.sortDirection === 'asc') {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, sorted by Date Added ascending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, sorted by %(sort_name)s ascending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, sorted by {sortName} ascending'
+                        );
                     } else {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, sorted by Date Added descending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, sorted by %(sort_name)s descending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, sorted by {sortName} descending'
+                        );
                     }
                 }
 
-                return '<p>' + interpolate(message, {
-                        current_item_range: this.currentItemRangeLabel(),
-                        total_items_count: this.totalItemsCountLabel(),
-                        asset_type: asset_type,
-                        sort_name: this.sortNameLabel()
-                    }, true) + "</p>";
+                return '<p>' + HtmlUtils.interpolateHtml(message, {
```

Comment:
In fact, I'd probably just return the message as HTML, and then let the Underscore template wrap it in a ``<p>`` tag or whatever it chooses.